internal/provider/provider.go

```diff
@@ -315,18 +315,26 @@ func Provider() *schema.Provider {
 				Description: "The secret key for API operations. You can retrieve this\n" +
 					"from the 'Security & Credentials' section of the AWS console.",
 			},
-			"shared_config_file": {
```

Comment:
should we consider deprecating this field first before full removal for consistency?

Reply:
ahaha ignore me..all good . the was added just recently in the milestone 🤦‍♀️